tests/projects/Sample_ConsoleApp_net7/Sample_ConsoleApp_net7.fsproj

```diff
@@ -8,6 +8,9 @@
 
   <PropertyGroup>
     <DotnetFscCompilerPath>$(MSBuildThisFileDirectory)../../../artifacts/bin/fsc/Debug/net6.0/fsc.dll</DotnetFscCompilerPath>
+    <Fsc_DotNET_DotnetFscCompilerPath>$(MSBuildThisFileDirectory)../../../artifacts/bin/fsc/Debug/net6.0/fsc.dll</Fsc_DotNET_DotnetFscCompilerPath>
```

Comment:
We should probably adjust the FSharp.Core.UnitTests and a few others to do this.  I've noticed they use the Proto to build, perhaps it's better to just use the built compiler.